src/services/upper-bar-manager/ui/displayed-bar/displayed-bar.component.tsx

```diff
@@ -85,7 +78,7 @@ export class DisplayedBar extends Component<DisplayedBarProps, DisplayedBarState
             </IconWrapper>
           );
         })}
-        {dropdownControls.length && <MoreIcon icons={dropdownControls} />}
+        {dropdownControls.length > 0 && <MoreIcon icons={dropdownControls} />}
```

Comment:
fix for "0" that renders in DOM
<img width="457" alt="Screenshot 2023-01-25 at 15 53 24" src="https://user-images.githubusercontent.com/51074448/214586438-05c27f49-33ec-43c6-8f0c-535581939c0f.png">